cradle-admin-tool-http/build.gradle

```diff
@@ -27,10 +27,6 @@ application {
     mainClassName = "com.exactpro.th2.cradle.adm.http.Application"
 }
 
-test {
-    useJUnitPlatform()
-}
```

Comment:
Could you let me know why you had to remove it?

Reply:
Got it. Missed that moment

Reply:
No, problem )